cluster/manifests/z-kubenurse/daemonset.yaml

```diff
@@ -0,0 +1,81 @@
+{{if eq .Cluster.ConfigItems.network_monitoring_enabled "true" }}
+---
+# Source: kubenurse/templates/daemonset.yaml
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  labels:
+    application: kubernetes
+    component: kubenurse
+  name: kubenurse
+  namespace: kubenurse
+spec:
+  selector:
+    matchLabels:
+      application: kubernetes
+      component: kubenurse
```

Comment:
let's use the `daemonset: kubenurse` label pattern for the matchLabels like we do for all other components.